cocktails9/cocktails9/ContentView.swift

```diff
@@ -6,7 +6,7 @@ struct ContentView: View {
     var body: some View {
         SplashView(navigateToLogin: $navigateToLogin)
```

Comment:
SplashView should know nothing about login or further navigation. It should just be pretty when its parent presents it and be replaced when parent decides. In this case, move presentation logic to ContentView.